source/packages/integration-tests/src/support/device_patcher_hooks.ts

```diff
@@ -45,12 +51,34 @@ function getAdditionalHeaders(world: unknown): Dictionary {
 }
 
 const templatesSvc: TemplatesService = container.get(DEVICE_PATCHER_CLIENT_TYPES.TemplatesService);
+const patchSvc: PatchService = container.get(DEVICE_PATCHER_CLIENT_TYPES.PatchService);
 
 async function deletePatchTemplate(world: unknown, name: string) {
     try {
         await templatesSvc.deleteTemplate(name, getAdditionalHeaders(world));
     } catch (e) {
-        logger.error(e);
+        logger.error(`deletePatchTemplate: ${JSON.stringify(e)}`);
+    }
+}
+
+async function deletePatchesForDevice(world: unknown, deviceId: string) {
+    try {
+        // get all patches for the given device
+        const patchList: ListPatchResponse = await patchSvc.listPatchesByDeviceId(
+            deviceId,
+            undefined,
+            getAdditionalHeaders(world)
+        );
+        const patches: PatchResponse[] = patchList.patches;
+
+        // delete all patches for the given device
+        const deletionPromises = patches.map((patch): Promise<void> => {
+            console.log(`deleting: ${patch.patchId}`);
```

Comment:
Is there a reason to use console.log?

Reply:
These are in the integ tests, so I don't think it matters either way, but I removed it.